### DISCUSSION TITLE
How to get channel ID from a video url?

### DISCUSSION THREAD

**Discussion by LinArcX** [pystardust/ytfzf#687] [Q&A] [answered]
I'm watching some videos and i'm interested to have the Channel name that starts with `@`, i couldn't find any option for that.

**Comment by Euro20179** [comment]
If you mean adding the channel to the subscriptions, you can use the @ version, eg: `@brodierobertson`

**Reply by LinArcX** [reply]
no. i mean finding chanel name.

**Reply by Euro20179** [reply, marked as answer]
Oh I see, no there's no way to get the @ name for a channel from the id in ytfzf.
